### PR TITLE
Rewrite of the documentation.

### PR DESCRIPTION
**Pull request by snappyapple632**
I've changed a lot of things throughout. Please check for any empty links or anything else I could have missed.

### CODE REVIEW

File: docs/06_documentation.md [hunk 1]
```diff
@@ -39,26 +39,26 @@ Also Linux/*BSD comes pre installed with python. Most distributions also deliver
...
+From herein, everything will be the same as on Desktop. If you want to edit the files, you can (besides vim or emacs which are available through Termux) use your preferred text editor on Android. This is possible by opening the files with the Termux integration of the build in android file manager:
 
 ![termux_files](img/termux_files.png)
 
-## Update
-Sometimes mkdocs changes the way how to serve, or the syntax will differ. This is why you should make sure to always run the latest version of mkdocs. To ensure this simply run `pip3 install --upgrade mkdocs` or `sudo pip3 install --upgrade mkdocs` if you installed pip system wide on a linux/bsd system.
+## Updating
+Sometimes, mkdocs changes the way of how it serves, or the syntax will differ. This is why you should make sure to always run the latest version of mkdocs. To check. simply run `pip3 install --upgrade mkdocs` or `sudo pip3 install --upgrade mkdocs` if you installed pip system wide on a Linux/BSD* system.
```
Comment (TobiGr) [inline docs/06_documentation.md]:
```suggestion
Sometimes, mkdocs changes the way of how it serves, or the syntax will differ. This is why you should make sure to always run the latest version of mkdocs. To check, simply run `pip3 install --upgrade mkdocs` or `sudo pip3 install --upgrade mkdocs` if you installed pip system wide on a Linux/BSD* system.
```

File: docs/06_documentation.md [hunk 2]
```diff
@@ -39,26 +39,26 @@ Also Linux/*BSD comes pre installed with python. Most distributions also deliver
...
-In order to extend this documentation you have to clone it from its [git repository](https://github.com/TeamNewPipe/documentation). When you cloned it, you will find a [mkdocs.yml](https://github.com/TeamNewPipe/documentation/blob/master/mkdocs.yml) file, and a [docs](https://github.com/TeamNewPipe/documentation/tree/master/docs) directory inside. The yaml file is the [config file](https://www.mkdocs.org/user-guide/configuration/) while in the directory docs the documentation files are stored. [here](https://www.mkdocs.org/user-guide/writing-your-docs/) is a guide about how to use mkdocs.
+In order to extend this documentation, you have to clone it from its [GitHub repository](https://github.com/TeamNewPipe/documentation). When you clone it, you will find a [mkdocs.yml](https://github.com/TeamNewPipe/documentation/blob/master/mkdocs.yml) file, and a [docs](https://github.com/TeamNewPipe/documentation/tree/master/docs) directory inside. The yaml file is the [config file](https://www.mkdocs.org/user-guide/configuration/) while in the directory docs the documentation files are stored. [here](https://www.mkdocs.org/user-guide/writing-your-docs/) is a guide about how to use mkdocs.
```
Comment (TobiGr) [inline docs/06_documentation.md]:
```suggestion
In order to extend this documentation, you have to clone it from its [GitHub repository](https://github.com/TeamNewPipe/documentation). When you clone it, you will find a [mkdocs.yml](https://github.com/TeamNewPipe/documentation/blob/master/mkdocs.yml) file, and a [docs](https://github.com/TeamNewPipe/documentation/tree/master/docs) directory inside. The yaml file is the [config file](https://www.mkdocs.org/user-guide/configuration/) while in the directory docs the documentation files are stored. [Here](https://www.mkdocs.org/user-guide/writing-your-docs/) is a guide about how to use mkdocs.
```

File: docs/index.md [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
+Please be aware that it is in its early stages, so help and [feedback](https://github.com/TeamNewPipe/documentation/issues) is always appreciated :D
 
 
 ## Introduction
 
-The NewPipeExtractor is a Java framework for scraping video platform websites in a way that they can be accessed like a normal API. The extractor is the core of the popular YouTube and streaming App [NewPipe](https://newpipe.schabi.org) for android, however it's system independent and also available for other platforms as well. 
+The NewPipeExtractor is a Java framework for scraping video platform websites in a way that they can be accessed like a normal API. The extractor is the core of the popular YouTube and streaming app [NewPipe](https://newpipe.schabi.org) for Android. It is entirely independent from said platforms and also available for additional platforms as well. 
 
-The beauty behind this framework is it takes care of the extracting process, error handling etc., so you can take care about what is important: Scraping the website.
+The beauty behind this framework is that it takes care of the extracting process, error handling etc. so you can focus on what is important: Enjoying the content.
```
Comment (TobiGr) [inline docs/index.md]: Why did you use _enjoying the content_ here? IMHO this does not match the original meaning of _scraping the content_.

File: docs/00_Prepare_everything.md [hunk 4]
```diff
@@ -1,86 +1,82 @@
...
 
-- NSFL (Not Safe For Life)
-- Porn that is not allowed according to [US Porn act](https://www.justice.gov/archive/opa/pr/2003/April/03_ag_266.htm).
-- Any form of violence
-- Child pornography
-- Media that harms others
-- Media that violates human rights
-- Copyright infringement/pirated media
+- Content that is considered NSFL (Not Safe For Life)
+- Content that is prohibited by US federal law (Sexualization of minors, etc).
```
Comment (TobiGr) [inline docs/00_Prepare_everything.md]: @TheAssassin or @theScrabi can you please take a look at this. I'd suggest to keep the content of the three bullet points.
```suggestion
- Content that is prohibited by US federal law (Sexualization of minors, any form of violence, violations of human rights, etc).
```

Comment (TobiGr) [inline docs/00_Prepare_everything.md]: @TheAssassin or @theScrabi can you please take a look at this? I'd suggest to keep the content of the three bullet points.
```suggestion
- Content that is prohibited by US federal law (Sexualization of minors, any form of violence, violations of human rights, etc).
```

File: docs/index.md (hunk 3 above)
Comment (theScrabi) [reply docs/index.md]: Na it must keep Scraping. Enjoying does not fit here. This sentense is not ment from the view of the user, but from the view of the developer. The developer should focus on what is important about what he does, and for him it is important to scrape the content.

**Comment by theScrabi** [comment]
@snappyapple632 CI failes, please make sure there are no dengling links in the documentation.

**Comment by snappyapple632** [comment]
Alright, I'll do another run through the docs tonight and fix the rest of the issues here. @theScrabi , can you provide the specific links that aren't working?

**Comment by TobiGr** [comment]
I don't think that it's your fault. As you can see [here](https://travis-ci.org/TeamNewPipe/documentation/builds), the link checks failed on the master branch, too.

**Comment by snappyapple632** [comment]
All the links are fixed now. It's ready for final review.

**Pull request merged**